goa/Classes/menu/MainMenuHome.cpp

```diff
@@ -194,15 +205,46 @@ void MainMenuHome::bindEvents(cocos2d::Node *rootNode)
                 if (textTitle)
                 {
                     std::string translatedString = this->currentLangUtil->translateString(textTitle->getName());
-                    textTitle->setText(translatedString);
+                    string gameName = textTitle->getName().c_str();
+                    string mainMenuEnglishText = "";
```

Comment:
Check if you can directly update the source

Reply:
If we use the text from the source then there is an alignment problem in the Hindi text.